src/utils/contract.ts

```diff
@@ -103,7 +103,7 @@ export default class Contract extends BaseConnection {
     if (this.provider) {
       return await this.provider
       .sendAsync('eth_call', params)
-      .then(resp => resp.json !== undefined ? resp.json() : resp);
+      .then(resp => resp.json());
```

Comment:
`JsonRpcProvider::send` responses don't have `json` method either. Seems like this code never worked even for ethers.js